docs/work-with-us/releases/versions-compatibility-matrix.md

```diff
@@ -1,12 +1,12 @@
 
 # Leverage Releases & Versioning
 
-Binbash Leverage™ and its components intends to be backward compatible, but due to the complex ecosystems of tools we 
-manage this is not always possible. 
+Binbash Leverage™ and its components intends to be backward compatible, but due to the complex ecosystems of tools we
+manage this is not always possible.
 
-It is always recommended using the latest version of the [Leverage cli](https://pypi.org/project/leverage/) with the latest versions of the 
-[Reference Architecture for AWS](https://github.com/binbashar/le-tf-infra-aws/releases). In case that's 
-not possible we always recommend pinning versions to favor stability and doing controlled updates 
+It is always recommended using the latest version of the [Leverage cli](https://pypi.org/project/leverage/) with the latest versions of the
```

Comment:
This is more a question for @exequielrafaela . Are we using the capitalized version of "CLI" word? I.e. "Leverage CLI" instead "Leverage cli". If this is true we can take advantage of this change and fix the word here.

Reply:
@juanmatias thanks for pointing this out. In most occurrences we're using "Leverage CLI", so yes we must keep this consisten all over the doc.